bokeh/core/validation/errors.py

```diff
@@ -38,10 +46,12 @@
     1001: ("BAD_COLUMN_NAME",        "Glyph refers to nonexistent column name"),
     1002: ("MISSING_GLYPH",          "Glyph renderer has no glyph set"),
     1003: ("NO_SOURCE_FOR_GLYPH",    "Glyph renderer has no data source"),
-    1004: ("REQUIRED_RANGE",         "A required Range objects missing"),
+    1004: ("REQUIRED_RANGE",         "A required Range object is missing"),
     1005: ("MISSING_GOOGLE_API_KEY", "Google now requires API keys for all Google Maps usage"),
     1006: ("NON_MATCHING_DATA_SOURCES_ON_LEGEND_ITEM_RENDERERS", "LegendItem.label is a field, but renderer data sources don't match"),
     1007: ("MISSING_MERCATOR_DIMENSION", "Mercator Tickers and Formatters must have their dimension property set to 'lat' or 'lon'"),
+    1008: ("REQUIRED_SCALE",         "A required Scale object is missing"),
```

Comment:
I'm unsure whether this should be it's own error or whether it could/should be folded into "INCOMPATIBLE_SCALE_AND_RANGE". The only time it'd ever come up (IIRC), is if both ranges and scales are set to None (overriding the default for the latter)